native/explorer/src/expressions.rs

```diff
@@ -602,7 +602,7 @@ pub fn expr_last(expr: ExExpr) -> ExExpr {
 
 #[rustler::nif]
 pub fn expr_format(exprs: Vec<ExExpr>) -> ExExpr {
-    ExExpr::new(concat_str(ex_expr_to_exprs(exprs), ""))
+    ExExpr::new(concat_str(ex_expr_to_exprs(exprs), "", true)) //TODO: ignore_nulls
```

Comment:
ignore_nulls need to be true is what I felt reading docs. let me know what if it needs to be false.

Reply:
I would like @cigrainger and @billylanchantin opinion on this one (and we should write tests).

Reply:
also should we add new param `ignore_nulls` just like polars ? I get confused about naming should it be `ignore_nils` or `ignore_nulls`.

Reply:
https://github.com/elixir-explorer/explorer/pull/861#discussion_r1493125106

Reply:
> also should we add new param `ignore_nulls` just like polars ?

I would be fine leaving this as a TODO for this PR. After reading:

* https://github.com/pola-rs/polars/pull/13877

I can see the value in having it. But the additional testing/documentation would necessitate a lot more work which we could instead do in a follow-up.

Reply:
I agree with @billylanchantin. We can do it in a follow-up.